README.md

```diff
@@ -1,134 +1,18 @@
-# CVMFS CSI driver
+# CVMFS CSI plugin
 
 [![Build Status](https://github.com/cernops/cvmfs-csi/workflows/csi-cvmfsplugin/badge.svg?event=push&branch=master)](https://github.com/cernops/cvmfs-csi/actions?workflow=csi-cvmfsplugin)
 [![Go Report Card](https://goreportcard.com/badge/github.com/cernops/cvmfs-csi)](https://goreportcard.com/report/github.com/cernops/cvmfs-csi)
 [![GoDoc](https://godoc.org/github.com/cernops/cvmfs-csi?status.svg)](https://godoc.org/github.com/cernops/cvmfs-csi)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-csi-cvmfs provides read-only mounting of CVMFS volumes in CSI-enabled container orchestrators.
+CVMFS [Container Storage Interface](https://github.com/container-storage-interface/spec) (CSI) plugin provides read-only mounting of [CernVM-FS](https://cernvm.cern.ch/fs/) (CVMFS) repositories in CSI-enabled container orchestrators.
 
-## Building
+See project documentation at [docs/](./docs).
 
-To build the binary in ./bin:
-```
-make build
-```
+## Reporting bugs
 
-To build binaries for multiple platforms:
-```
-make build-cross
-```
+Please report issues at https://github.com/cernops/cvmfs-csi/issues/new.
 
-To make the docker image:
-```
-make docker
-```
+## Contributing
 
-## Configuration
```

Comment:
Must these documentation sections be lost?  Can they be updated for the new setup?

Reply:
@jcpunk They are not lost, only moved to separate documents under docs/. Does this work for you, or did we miss something?

Reply:
Ahh, I missed those.  Can you add a link in the README to point there?

Reply:
There already is, https://github.com/cvmfs-contrib/cvmfs-csi/pull/39/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R10 :) 

https://github.com/cvmfs-contrib/cvmfs-csi/blob/0b27377259a716e7f68a3b1a9dd3c890d8fcbf50/README.md